pandoc.cabal

```diff
@@ -560,6 +538,8 @@ library
                    Text.Pandoc.MediaBag,
                    Text.Pandoc.Error,
                    Text.Pandoc.Filter,
+                   Text.Pandoc.Filter.Environment,
+                   Text.Pandoc.Filter.JSON,
```

Comment:
Maybe the relevant functions should be exported by `Text.Pandoc.Filter`, so these submodules can stay internal?

Reply:
Yes, I think that makes sense.